### ISSUE TITLE
I Cant open this mod

### ISSUE THREAD

**Issue by chairfreaker60** [UnderminersTeam/UndertaleModTool#1463]
### Describe the bug

![Screenshot 2023-08-21 150531](https://github.com/krzys-h/UndertaleModTool/assets/135190287/ac1a6ea2-f56e-45a8-b188-ff809c97ad8f)
![Screenshot 2023-08-21 150544](https://github.com/krzys-h/UndertaleModTool/assets/135190287/8d908335-7e36-4c07-91b7-46f37b8a67b8)

### Reproducing steps

Its probably related more to the mod, its probably nothing to do with UMT, but i feel like it should be looked at anyway. The mod was for an older version of undertale, idk if this supports the old version

### Setup Details

basically its an underfell mod, it only works on the old version of undertale, and it is only a reskin, i wanna increase the difficulty, so i open this and it happened

**Comment by Miepee** [comment]
Probably related to #855

**Comment by Miepee** [comment]
@chairfreaker60 do you have a link to the mod, so that we can repdouce it?

**Comment by Miepee** [comment]
Closing due to being inactionable.
Feel free to reopen the issue when you provide more info.
